.github/workflows/lint.yaml

```diff
@@ -12,3 +12,15 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-features
+  loc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install cargo-count
+        run: cargo install cargo-count
+
+      - name: Run cargo count
+        # '-a --exclude=$(cat .gitignore)' necessary to work around:
+        # https://github.com/kbknapp/cargo-count/issues/36
+        run: cargo count -l rs -a --exclude=$(cat .gitignore) src
```

Comment:
nit: missing newline :)

Reply:
I can add it, but Github Actions seems to be fine with it, necessary?

Reply:
Not necessary but it's good to always have the trailing newline (some editors and CLI tools don't like it when the newline is missing).